src/components/HomePage.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Route, Routes } from 'react-router';
```

Comment:
You successfully created a single page application (SPA) using react-router-dom to serve each page without reloading 🙌🔥